README.md

```diff
@@ -419,7 +419,46 @@ On Linux, it is often necessary to first update `pip` to a version that supports
 
 ### pip installation: GPU (CUDA)
 
-If you want to install JAX with both CPU and NVidia GPU support, you must first
+JAX provides pre-built CUDA wheels for **Linux only**. Other
```

Comment:
nit: "Installing on other operating systems is possible, ..."

Reply:
Done.